docs/installation/systemd.md

```
@@ -236,6 +237,37 @@ instructions.
 > systemd clusters. As with other port fields, you can specify `0` to
 > assign a random port.
 
+## Performing Postgres Minor Version Upgrades
+
+Database upgrades are not yet supported via the Control Plane API, but system
+administrators can perform minor Postgres version upgrades by updating the
+packages on each machine. Follow these steps on each host in the cluster:
+
+1. Upgrade Postgres and/or other components using `dnf upgrade`. For example:
+
+    ```sh
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)